pynucastro/library/tabular/README.md

```diff
@@ -10,6 +10,7 @@ rates are tabulated in a two-dimensional space of:
 This directory includes tabulated rates obtained from Suzuki et al.,
 2016, ApJ 817:163, downloaded from
 https://www.phys.chs.nihon-u.ac.jp/laboratory/suzuki
+https://web.archive.org/web/20210125194952/https://www.phys.chs.nihon-u.ac.jp/laboratory/suzuki
```

Comment:
A still-working english link is https://www.phys.chs.nihon-u.ac.jp/suzuki/data2/